<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="o_graph" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>801</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="n_graph" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>620</y>
      <width>801</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(177, 177, 177);
border: 1px solid black;
border-radius: 4px;
font: 18pt &quot;Nanum Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(177, 177, 177);
border: 1px solid black;
border-radius: 4px;
font: 18pt &quot;Nanum Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="front_cam">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>08SeoulNamsan</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 24pt &quot;08SeoulNamsan&quot;;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Front-Cam</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="side_cam">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>08SeoulNamsan</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 24pt &quot;08SeoulNamsan&quot;;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>Side-Cam</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="o_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
color: rgb(0, 0, 0);
font: 24pt &quot;Nanum Gothic&quot;;
background-color: rgb(255, 255, 255);
border-radius: 3px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;AttenD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="n_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 24pt &quot;Nanum Gothic&quot;;
border-radius: 3px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;개선 방식&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="o_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
font: 18pt &quot;Nanum Gothic&quot;;
</string>
    </property>
    <property name="text">
     <string>Driving Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="n_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>690</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
font: 18pt &quot;Nanum Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Driving Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="activity">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 18pt &quot;Nanum Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Activity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gaze">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 18pt &quot;Nanum Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Gaze</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="activity_title">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 16pt &quot;Nanum Gothic&quot;;
border-radius: 3px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>행동</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="gaze_title">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
font: 16pt &quot;Nanum Gothic&quot;;
background-color: rgb(255, 255, 255);
border-radius: 3px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>시선</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="driving">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
font: 18pt &quot;Nanum Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Driving</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="driving_title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 18pt &quot;Nanum Gothic&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>운전 상황</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
